bin/pycbc_inspiral_skymax

```diff
@@ -210,7 +224,22 @@ with ctx:
         logging.info("Finished")
         sys.exit(0)
 
-    if opt.sky_maximization_method == 'hom':    
+    # FIXME: Maybe we should use the PSD corresponding to each trigger
+    if opt.psdvar_segment is not None:
+        logging.info("Calculating PSD variation")
+        psd_var = pycbc.psd.calc_filt_psd_variation(gwstrain, opt.psdvar_segment,
+                opt.psdvar_short_segment, opt.psdvar_long_segment,
+                opt.psdvar_psd_duration, opt.psdvar_psd_stride,
+                opt.psd_estimation, opt.psdvar_low_freq, opt.psdvar_high_freq)
+
+    if opt.enable_q_transform:
+        logging.info("Performing q-transform on analysis segments")
+        q_trans = qtransform.inspiral_qtransform_generator(segments)
```

Comment:
As far as I can see this function doesn't exist??

Reply:
Looks like it was remove some time ago, probably my mistake for not catching to remove it here as well. I don't remember the history here.

Reply:
https://github.com/gwastro/pycbc/blob/master/bin/pycbc_inspiral#L291

I don't think this was removed from the main pycbc_inspiral.

Reply:
(Maybe that's what you meant, and I'm just a bit confused this morning :-) )

Reply:
@spxiwh No, I agree, it seems to have been left in the exes. I think the code it relies upon was removed though at some point. The underlying code used to exists, it was most likely removed in a shuffling by me, but I don't really remember what the thought process was then. At this point though, these lines are dead code and should just be removed.